chain/network/src/state_witness.rs

```diff
@@ -24,11 +24,11 @@ pub struct PartialEncodedStateWitnessForwardMessage(pub PartialEncodedStateWitne
 #[rtype(result = "()")]
 pub struct ChunkContractAccessesMessage(pub ChunkContractAccesses);
 
-/// Message to partial witness actor (on a chunk validator) that contains code-hashes of
-/// the contracts that are deployed when applying the previous chunk.
+/// Message to partial witness actor (on a chunk validator) that contains part of
+/// code-hashes of the contracts that are deployed when applying the previous chunk.
```

Comment:
not code-hashes, but complete contract code

Reply:
done